api-service/src/main/java/com/bitmark/apiservice/middleware/Converter.java

```diff
@@ -84,7 +85,8 @@ public void onSuccess(Response response) {
                 try {
                     String raw = response.body().string();
                     Map<String, Object> json = jsonToMap(raw);
-                    callback.onSuccess(json.get("txid").toString());
+                    String txId = json.get("txid") != null ? json.get("txid").toString() : json.get("txId").toString();
```

Comment:
It's fine but i think `json.get("txId").toString()` might provide `NullPointerException`. Could you check for specific case and return empty string if nothing matches.

Reply:
In this case, it should throw Exception if it doesn't contain either txid or txId

Reply:
OK